### PR TITLE
[reactstrap]: Export Util.setGlobalCssModule

### PR DESCRIPTION
**Pull request by kyletsang**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/reactstrap/reactstrap/issues/1789
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

As per discussion in https://github.com/reactstrap/reactstrap/issues/1789, `setGlobalCssModule` should be exposed.  As for the others, I believe it should be added on a case by case basis?

### CODE REVIEW

File: types/reactstrap/index.d.ts [hunk 1]
```diff
@@ -193,3 +193,5 @@ export {
   UncontrolledCarouselProps,
   UncontrolledPopoverProps
 } from './lib/Uncontrolled';
+import * as Util from './lib/Util';
```
Comment (PranavSenthilnathan) [inline types/reactstrap/index.d.ts]: `export { Util } from './lib/Util':` to match the style in the file?

Comment (kyletsang) [reply types/reactstrap/index.d.ts]: reactstrap exports the functions inside Util with `export * as Util from './utils'` but it seems this is not available until Typescript 3.8. 

Would it be better to put the functions in Util under a namespace and export that?

Util.d.ts
```
declare namespace Util {
    export function setGlobalCssModule(cssModule: CSSModule): void;
}

export default Util;
```

index.d.ts
```
export { default as Util } from './lib/Util';
```

Comment (sheetalkamat) [reply types/reactstrap/index.d.ts]: This looks ok

File: types/reactstrap/lib/Util.d.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import { CSSModule } from '../index';
```
Comment (sheetalkamat) [inline types/reactstrap/lib/Util.d.ts]: Can you please name this file utils.d.ts instead to match the lib structure in the package.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

**Comment by kyletsang** [comment]
I've made the file name change. Thanks

**Pull request merged**